app/containers/HeaderButton/HeaderButtonItem.js

```diff
@@ -32,7 +33,12 @@ const styles = StyleSheet.create({
 const Item = ({
 	title, iconName, onPress, testID, theme, badge
 }) => (
-	<Touchable onPress={onPress} testID={testID} hitSlop={BUTTON_HIT_SLOP} style={styles.container}>
+	<Pressable
```

Comment:
Does this change impacts other places?
Is it working correctly on both OS?
Is badge working?